packages/reassure-cli/src/commands/check-stability.ts

```diff
@@ -1,13 +1,18 @@
 import type { CommandModule } from 'yargs';
+
 import { applyCommonOptions, CommonOptions } from '../options';
-import { configureLoggerOptions } from '../utils/logger';
+import { bye, configureLoggerOptions, hello } from '../utils/logger';
 import { run as measure } from './measure';
 
 export async function run(options: CommonOptions) {
   configureLoggerOptions(options);
 
-  await measure({ ...options, baseline: true });
-  await measure({ ...options, baseline: false, compare: true });
+  hello(options);
+
+  await measure({ baseline: true, isSubRoutine: true });
```

Comment:
1. Why are options no-longer passed to `measure`?
2. `isSubRoutine` name is not hard to understand without context.